package.json

```diff
@@ -84,8 +84,10 @@
   "types": "types/index.d.ts",
   "files": [
     "dist",
-    "types",
-    "index.d.ts"
+    "types"
   ],
+  "overrides": {
+    "rollup": "npm:@rollup/wasm-node"
```

Comment:
Can you elaborate on why this is necessary?

Reply:
Its a weird bug in npm, see my first comment. But its to do with the test framework using rollup, if you look at the logs of one of the early failed builds, it complains it cant find the native rollup.

The fix is to add an override to package.json, as seen here: https://github.com/vitejs/vite/issues/15167

Reply:
Specifically this run: https://github.com/jitsi/lib-jitsi-meet/actions/runs/7251356239/job/19753537267